src/server/game/Accounts/RBAC.h

```diff
@@ -748,7 +748,14 @@ enum RBACPermissions
     RBAC_PERM_COMMAND_NEARGRAVEYARD                          = 841,
     RBAC_PERM_COMMAND_RELOAD_CHARACTER_TEMPLATE              = 842,
     RBAC_PERM_COMMAND_RELOAD_QUEST_GREETING                  = 843,
-    RBAC_PERM_COMMAND_DEBUG_SEND_PLAYSCENE                   = 844,
+    RBAC_PERM_COMMAND_DEBUG_SEND_PLAYSCENE                   = 844, // deprecated
```

Comment:
I did in the merge, see https://github.com/TrinityCore/TrinityCore/commit/d4887311e39527e036419ebd966f399fdbdae67a#diff-ff3b59d5f6fc8303cc01120ec5c190fcR751